gtdynamics/optimizer/ConstrainedOptimizer.h

```diff
@@ -30,6 +30,13 @@ struct ConstrainedOptimizationParameters {
   ConstrainedOptimizationParameters() {}
 };
 
+/// Intermediate results for constrained optimization process.
+struct ConstrainedOptResult {
+  std::vector<gtsam::Values> values_list;  // values after each inner loop
```

Comment:
never put type in a variable name, and definitely not the wrong type :-) Just say `intermediate_values`

Reply:
Done